WordPress/gutenberg
# Unify the global block styles panel with the block inspector panels

closes #47348 

## What?

The Global Styles panel for blocks is very different than the block inspector for the same block. There's no clear reason for this difference. This PR brings them close together as per the designs in #47348 

## Todo:

 - [x] Add Style Variations support back
 - [x] Add Effects panel back #49571
 - [x] Add Filters panel back #49577 
 - [x] Add custom CSS panel back
 - [x] Figure out what to do with the color palettes per block (not visible in the current design)


## Testing Instructions

1- Open the global styles for blocks (paragraph, image,...)
2- Make sure the design match the issue and work as expected.
